app/services/builders/article_params_builder.rb

```diff
@@ -0,0 +1,34 @@
+class ArticleParamsBuilder
```

Comment:
Have some adjustments to your builder, overall good but more like a different approach for your decision 
```ruby
class ArticleParamsBuilder
  # Use author instead author_id it looks simplier and more clear and it's ruby you can pass objects
  def initialize(params, author)
    @params = params
    @author = author
  end

  def build
    article_params.merge(article_translations_attributes: article_translation_params)
  end

  private

  attr_reader :params, :author

  def article_params
    {
      category_id:   params[:category_id],
      conference_id: params[:conference_id],
      team_id:       params[:team_id],
      location_id:   params[:location_id],
      author_id:     author.id
    }
  end

  def article_translation_params
    {
      language_id:   params[:language_id],
      picture:       params[:picture],
      alt_image:     params[:alt_image],
      headline:      params[:headline],
      caption:       params[:caption],
      content:       params[:content],
      show_comments: sanitize_bool(params[:show_comments]),
      status:        Article::UNPUBLISHED
    }
  end

  def sanitize_bool(value)
    value == "1" ? true : false
  end
end
```

Reply:
Looks awesome